Trustroots/trustroots
# Remove/angular directive host offer

#### Proposed Changes

* Use existing React component for mobile view for hosting info on profiles
* Remove now un-used Angular directive
    ![image](https://user-images.githubusercontent.com/87168/110224089-5264a100-7ee1-11eb-9e9a-533c1998a94f.png)


#### Testing Instructions

* Open user's own profile in mobile view, open hosting info
* Open other user's profile in mobile view, open hosting info
* Note that the dropdown doesn't work in current `master` (same issue as with language dropdown I assume)
* Everything should be identical as in production otherwise

<img width="375" alt="Screenshot 2021-03-07 at 01 04 27" src="https://user-images.githubusercontent.com/87168/110224068-26492000-7ee1-11eb-9989-cc1ad437ffdf.png">
